### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by tychop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,18 @@ By default, ```sigh``` will install the downloaded profile on your machine. If y
 
     sigh --skip_install
 
+If you need the provisioning profile to be renewed regardless of it's state use the --force option. This gives you a profile with the maximum lifetime:
+
+    sigh --renew --force -a com.krausefx.app -u username
```
Comment (KrauseFx) [inline README.md]: What does `--renew` do? I think that should be removed, right?

Comment (tychop) [inline README.md]: Whoops. That should actually be just renew instead of --renew...
--force without the renew just downloads the profile.

**Comment by tychop** [comment]
You are correct. Renew isn't needed at all.. Somehow I've tested everything with renew... But I now see it works without as well.

**Comment by KrauseFx** [comment]
Thank you @tychop :+1:

**Pull request merged**